.github/workflows/build.yml

```
@@ -0,0 +1,34 @@
+name: Build Fobnail binaries
+
+on:
+  push:
+
+env:
+  FOBNAIL_SDK_VERSION: v0.2.3
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

`actions/checkout` has support for cloning submodules
```suggestion
        uses: actions/checkout@v2
        submodules: recursive
```

---

Please try that and if it works we can use that. I copied it over from another project, where there were some isues with the builtin submodule clone.

---

Done